.circleci/config.yml

```
@@ -221,8 +221,8 @@ jobs:
             tar xfz julia-*-linux-x86_64.tar.gz
             ln -s $(pwd)/julia-*/bin/julia /usr/bin/julia
             # conda update -y conda > /logs/conda.txt 2>&1
-            # Create and activate conda environment
-            conda create -y --name esmvaltool
+            # Create and activate conda environment; install latest python in it
+            conda create -y --name esmvaltool python=3.9.2 -c conda-forge
             set +x; conda activate esmvaltool; set -x
             # Install
             conda install -y esmvaltool -c conda-forge -c esmvalgroup
```

```suggestion
            # Install
            conda create -n esmvaltool -c conda-forge -c esmvalgroup esmvaltool
            set +x; conda activate esmvaltool; set -x
```
because that is how we install people to install this in the documentation. Hopefully that solves the issue too.

---

This tests is supposed to test that the instructions here work: https://docs.esmvaltool.org/en/latest/quickstart/installation.html#conda-installation

Maybe some of the other commands in this test could also be updated to more closely follow the instructions in the documentation

---

Bouwe, we're not installing people, people are beings not needing installation :rofl: I have the changes to the documentation ready to push but I wanted to see the test pass on CI first w/o retriggering it via a commit (plus I need to remove it from the habitual tests list anyway). Looks like the python forcing does the trick on CI too, since it's now installing the R packages (yay! but darn it's slooow)

---

and no, the way it's done in the documentation is not working w/o python forcing - @ledm tried the instructions yesterday and the whole show was not solving the env at all

---

huzzah! conda install test ran fine https://app.circleci.com/pipelines/github/ESMValGroup/ESMValTool/4456/workflows/ad52da5e-5842-4e1e-aa65-220929c8fa3f/jobs/43440

---

a side comment: the install instruction from the documentation is just a contracted form of the same workflow that we pass to the CI `conda_install` and the reason why both were not working is that an empty env created with miniconda3 contains python2.7 (don't ask me why, it's a bit silly, prob we should report this to the conda people). It's possible that now that Python diverged so much from 2.7 we've hit the wall where installing esmvaltool in that python2.7 environment is not possible anymore since conda reached its limit at upgrading the whole thing from 2.7 to 3.9 without taking crazy mmemory and spending ages

---

>  the reason why both were not working is that an empty env created with miniconda3 contains python2.7

This is why we no longer recommend installing esmvaltool into an existing environment in the documentation, but rather creating a new environment with esmvaltool in it with the command:
```
conda create -n esmvaltool -c conda-forge -c esmvalgroup esmvaltool
```
Note that this command does not first create an empty environment and then tries to upgrade it so esmvaltool can be installed. The command on CircleCI still creates an empty environment and then tries to install esmvaltool into that, so that's why I proposed changing it as shown above.

---

that didn't work for @ledm but if it works for you and it works on Circle then I'm all up for it! tell you what - how about you open a PR with that, and I change to Python 3.9 here as per your suggestion below, then we see which one work fine on Circle, if both work we'll go with your solution, how's that sound?